stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/Filters.kt

```diff
@@ -67,12 +67,21 @@ public object Filters {
     @JvmStatic
     public fun and(vararg filters: FilterObject): FilterObject = AndFilterObject(filters.toSet())
 
+    @JvmStatic
+    public fun and(filters: List<FilterObject>): FilterObject = AndFilterObject(filters.toSet())
+
     @JvmStatic
     public fun or(vararg filters: FilterObject): FilterObject = OrFilterObject(filters.toSet())
 
+    @JvmStatic
+    public fun or(filters: List<FilterObject>): FilterObject = OrFilterObject(filters.toSet())
```

Comment:
Do we really need all those new methods?
If anyone wants to use our methods with a list, they can always use the [spread operator](https://kotlinlang.org/docs/functions.html#variable-number-of-arguments-varargs)

```
val listOfFilters = listOf<FilterObject>(...)
val orFilter = or(*listOfFliters.toTypedArray())
```